app/src/main/java/de/tum/in/tumcampusapp/component/ui/news/KinoViewModel.kt

```diff
@@ -56,7 +56,7 @@ class KinoViewModel(private val localRepository: KinoLocalRepository,
                         .doOnError { Utils.log(it) }
                         .flatMapIterable { it }
                         .filter { it.isFutureMovie() }
-                        .subscribe { localRepository.addKino(it) }
+                        .subscribe({ localRepository.addKino(it) }, { t -> Utils.log(t) })
```

Comment:
we also have a `.doOnError { Utils.log(it) }` here...

Reply:
See reasoning above ☝️